wiki/Accessibility-A11y-Guide.md

```diff
@@ -8,6 +8,7 @@
   - [Using a11y scanner in android](#using-a11y-scanner-in-android)
   - [Using Talkback in android](#using-talkback-in-android)
   - [Useful Resources](#useful-resources)
+  - [Developers Videos](#developers-videos)
```

Comment:
My preference would be to add developer videos and slides under the related sections as opposed to creating a seperate section for them, but I will defer to @seanlip for Wiki document structure.

Reply:
Yes, first I thinking to add under resources section but then I think 1 problem is most of new contributors not read wiki completely, but everyone see TOC so when they see Developers Videos in ToC they directly jump to that instead of reading whole wiki. That's why I put it here. 
But yes @seanlip have more idea about structure.

Reply:
I suggest putting them in both places.

Reply:
Okay sure

Reply:
Done.